modules/functional-test/src/main/java/com/intuit/wasabi/tests/service/experiment/PaginationTest.java

```diff
@@ -85,62 +89,62 @@ public void t_ExperimentPaginationSmokeByCreationTime() {
     }
 
     @Test(dependsOnGroups = {"pagination_smoke"}, groups = {"pagination_pages"})
-    public void t_ExperimentPaginationNonExistingLatePage() {
+    public void experimentPaginationNonExistingLatePage() {
         List<Experiment> results = getPaginatedExperiments(4, 10, "experiment_name", null);
 
-        ModelAssert.assertEqualModelItems(results, Collections.<Experiment> emptyList());
+        ModelAssert.assertEqualModelItems(results, Collections.emptyList());
     }
 
     @Test(dependsOnGroups = {"pagination_smoke"}, groups = {"pagination_pages"})
-    public void t_ExperimentPaginationNonExistingNegativePage() {
+    public void experimentPaginationNonExistingNegativePage() {
         List<Experiment> results = getPaginatedExperiments(-4, 10, "experiment_name", null);
 
-        ModelAssert.assertEqualModelItems(results, Collections.<Experiment> emptyList());
+        ModelAssert.assertEqualModelItems(results, Collections.emptyList());
     }
 
     @Test(dependsOnGroups = {"pagination_smoke"}, groups = {"pagination_pages"})
-    public void t_ExperimentPaginationEmptyPage() {
+    public void experimentPaginationEmptyPage() {
         List<Experiment> results = getPaginatedExperiments(1, 0, "experiment_name", null);
-        ModelAssert.assertEqualModelItems(results, Collections.<Experiment> emptyList());
+        ModelAssert.assertEqualModelItems(results, Collections.emptyList());
     }
 
     @Test(dependsOnGroups = {"pagination_smoke"}, groups = {"pagination_pages"})
-    public void t_ExperimentPaginationFilterByName() {
+    public void experimentPaginationFilterByName() {
         List<Experiment> results = getPaginatedExperiments(1, 10, "experiment_name", "experiment_name=5");
 
         ModelAssert.assertEqualModelItems(results, experimentList.subList(5, 6));
     }
 
     @Test(dependsOnGroups = {"pagination_smoke"}, groups = {"pagination_pages"})
-    public void t_ExperimentPaginationFilterByIllegalKey() {
+    public void experimentPaginationFilterByIllegalKey() {
         String exception = getPaginationException(1, 10, "experiment_name", "illegal_key=some_value");
 
         Assert.assertTrue(exception.contains("illegal_key"));
     }
 
     @Test(dependsOnGroups = {"pagination_smoke"}, groups = {"pagination_pages"})
-    public void t_ExperimentPaginationSortByIllegalKey() {
+    public void experimentPaginationSortByIllegalKey() {
         String exception = getPaginationException(1, 10, "non_existent_sortorder", null);
```

Comment:
I am getting an exception in this test: java.lang.IllegalArgumentException: Cannot get property 'message' on null object